Transbank/Transbank.csproj

```diff
@@ -16,7 +16,7 @@
     <NeutralLanguage>en-US</NeutralLanguage>
     <PackageTags>payments, cl, chile, transbank</PackageTags>
     <PackageIconUrl>http://www.transbankdevelopers.cl/library/img/favicon/mstile-310x310.png</PackageIconUrl>
-    <VersionPrefix>1.3.1</VersionPrefix>
+    <VersionPrefix>2.0.1</VersionPrefix>
```

Comment:
@afiebig this shouldn't be 2.0.0 instead of 2.0.1?

Reply:
Nop, according to the readme, you have to put x.y.{z+1} in order to get the next pre-released generated after the x.y.z release.